runtime/doc/lsp.txt

```diff
@@ -2179,12 +2179,15 @@ stylize_markdown({bufnr}, {contents}, {opts})
     Return: ~
         (`table`) stripped content
 
-symbols_to_items({symbols}, {bufnr})         *vim.lsp.util.symbols_to_items()*
+                                             *vim.lsp.util.symbols_to_items()*
+symbols_to_items({symbols}, {bufnr}, {offset_encoding})
     Converts symbols to quickfix list items.
 
     Parameters: ~
-      • {symbols}  (`lsp.DocumentSymbol[]|lsp.SymbolInformation[]`)
-      • {bufnr}    (`integer?`)
+      • {symbols}          (`lsp.DocumentSymbol[]|lsp.SymbolInformation[]`)
+      • {bufnr}            (`integer?`)
+      • {offset_encoding}  (`'utf-8'|'utf-16'|'utf-32'?`) default to first
```

Comment:
This isn't very clear. First client according to what ordering?

Either way, a better phrasing might be: "Defaults to the encoding used by the first client in the buffer"

Reply:
This is to correspond with the previous `locations_to_items`. Should I synchronize the documentation of these two functions?

Reply:
I see. @lewis6991 I noticed that you last modified this description in https://github.com/neovim/neovim/commit/cbc82011ce19e308f249e1e3ba16f7b1219c9984. Are you okay with using the description from `character_offset`?
https://github.com/neovim/neovim/blob/359763307546ae7a401e9458fefeb6f4cd37a40f/runtime/doc/lsp.txt#L1905-L1906